src/app/features/fss-search/fss-search.component.ts

```diff
@@ -67,7 +69,10 @@ export class FssSearchComponent implements OnInit {
     private fssSearchValidatorService: FssSearchValidatorService,
     private fssSearchGroupingService: FssSearchGroupingService,
     private fssPopularSearchService: FssPopularSearchService,
-    private analyticsService: AnalyticsService) { }
+    private analyticsService: AnalyticsService) { 
+      this.displayPopularSearch = AppConfigService.settings["fssConfig"].displayPopularSearch;
+      console.log(this.displayPopularSearch);
```

Comment:
Please remove the `console.log()`

Reply:
Hi,
Removed the console.log. Also, resolved the input focus on field when popular search is selected.